doc/develop/sca/codechecker.rst

```diff
@@ -42,6 +42,23 @@ To configure CodeChecker or analyzers used, arguments can be passed using the
     -DCODECHECKER_ANALYZE_OPTS="--config;$CODECHECKER_CONFIG_FILE;--timeout;60"
 
 
+Storing CodeChecker results
+***************************
+
+If a CodeChecker server is active, the results can be uploaded using the optional
+``CODECHECKER_STORE`` or ``CODECHECKER_STORE_OPTS`` parameters, e.g.
+
+.. code-block:: shell
+
+    west build -b mimxrt1064_evk -s samples/basic/blinky -- -DZEPHYR_SCA_VARIANT=codechecker \
+    -DCODECHECKER_STORE_OTPS="--name;build;--url;localhost:8001/Default"
+
+.. note::
+
+    A name parameter is required and has to be passed to either ``CODECHECKER_ANALYZE_OPTS`` or
```

Comment:
I think the behavior when using `-DCODECHECKER_STORE=1` should be that it just works. Why not have a default name set during the analyze step? It'd be great to default to `${PROJECT_NAME}` but for some reason it seems empty, and I don't know CMake well enough to understand why :|

Reply:
I've defaulted to `zephyr`, not sure if there are better names available from `cmake` variables too..